tests/test_autoreport.py

```diff
@@ -359,3 +360,21 @@ def test_disaggregate(self):
                                    'stdev': u'*'}))})]
         for (i, stat) in enumerate(stats):
             assert stat == expected[i]
+
+    def test_disaggregate_extended_fields(self):
+
+        title, schemas, submissions = build_fixture('auto_report_extended_fields')
+        fp = FormPack(schemas, title)
+
+        report = fp.autoreport()
+        stats = report.get_stats(submissions, split_by="when")
+
+        assert stats.submissions_count == 22
+
+        stats = [(unicode(repr(f)), n, d) for f, n, d in stats]
```

Comment:
I know the other tests use this, but it's pretty confusing to me. Poking around in a debugger, I guess `f` is field, `n` is question name, and `d` are the statistics, in the form of a dictionary with keys `[u'values', u'provided', u'show_graph', u'not_provided', u'total_count']`?

Using more descriptive variables would set a good precedent for future tests.